src/screens/FeedScreen/index.js

```diff
@@ -248,7 +241,6 @@ const FeedScreen = (props) => {
         ref={listRef}
         contentHeight={dimen.size.FEED_CURRENT_ITEM_HEIGHT + normalizeFontSizeByWidth(4)}
         data={feeds}
-        onEndReach={onEndReach}
         onRefresh={onRefresh}
         onScroll={handleScrollEvent}
         onScrollBeginDrag={handleOnScrollBeginDrag}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` prop has been removed from the `TiktokScroll` component. As mentioned earlier, ensure that the removal of this prop and the addition of the `fetchNextFeeds` function on the `onMomentumScrollEnd` event provide the same user experience of infinite scrolling.



<!-- This is an auto-generated comment by OSS CodeRabbit -->